catalog/app/utils/AWS/Credentials.js

```diff
@@ -72,12 +72,14 @@ function useCredentialsMemo({ local }) {
 
 const Ctx = React.createContext()
 
-export function Provider({ children }) {
+export function AWSCredentialsProvider({ children }) {
   const cfg = Config.use()
   const local = cfg.mode === 'LOCAL'
   return <Ctx.Provider value={useCredentialsMemo({ local })}>{children}</Ctx.Provider>
 }
 
-export const useCredentials = () => React.useContext(Ctx)
+export function useCredentials() {
+  return React.useContext(Ctx)
+}
 
-export const use = useCredentials
+export { AWSCredentialsProvider as Provider, useCredentials as use }
```

Comment:
this would read better with one export per line? so maybe:
```
export {
  AWSCredentialsProvider as Provider,
  useCredentials as use
}
```

Reply:
i don't care about this much, just pointing out a potential style issue.

Reply:
i agree, but `prettier` doesnt -- it thinks the line is quite short as it is